shortex.tex

```diff
@@ -1,188 +1,622 @@
+\verb!$...$!, \verb!$$...$$!, 
+\verb!\begin{align}...\end{align}!, 
+\verb!\begin{aligned}...\end{aligned}!, 
+\verb!\begin{equation}...\end{equation}!, 
+not to mention starred versions of those environments 
+and \verb!\nonumber!/\verb!\notag! commands, depending 
+on whether/where you want equation numbers,
+
+The \texttt{autonum} package automatically decides which equations to provide
+numbers based on \textit{which equations you reference}. So when using ShorTeX,
+you only need two commands for math mode: single dollar signs \verb!$...$! for
+inline math, and \texttt{align} environments (redefined in ShorTeX to be
+\verb!\[...\]!) for display math.\footnote{There are
+differences between how \texttt{align} and 
+other math display environments typeset equations. I have not ever
+encountered a case where it mattered much. If you are very picky about typesetting,
+note that ShorTeX does not \emph{disable} any functionality, so you 
+can use other environments anywhere you feel it is necessary.}
+
+For example, if you create 
+the following display math,
+\begin{verbatim}
+\[
+   a+b = c \label{eq:the_equation}
+\]
+\end{verbatim}
+then if you use the command \verb!\cref{eq:the_equation}! somewhere
+in the document, that equation will automatically be assigned a number. If not, it
+won't get a number. See the \texttt{autonum} package 
+documentation \url{https://ctan.org/pkg/autonum?lang=en} for more information.
+
+
+
+\section{Custom macros}
+
+\subsection{Shrinking whitespace in math}
+The command \verb!\squish{<frac>}! in math mode enables you to shrink whitespace in mathematics,
+where \verb!<frac>! represents the fraction of whitespace reduction.
+Below, the first line is regularly spaced, the second line has \verb!\squish{0.5}!, and the third has \verb!\squish{0.0}!.
+\[
+	\sqrt{\frac{1^{2}}{0.111222}(0.111222\times1.111163+0.066987^{2}\times0.111222)-1}&= \sqrt{0.111222}\\
+	\squish{0.5}\sqrt{\frac{1^{2}}{0.111222}(0.111222\times1.111163+0.066987^{2}\times0.111222)-1}&= \sqrt{0.111222}\\
+	\squish{0.0}\sqrt{\frac{1^{2}}{0.111222}(0.111222\times1.111163+0.066987^{2}\times0.111222)-1}&= \sqrt{0.111222}\\
+\]
 
+The code for \verb!\squish! was taken from \url{https://tex.stackexchange.com/questions/467942/how-to-squeeze-a-long-equation}.
 
-\section{$\Lp{p}$ Spaces and Operators}
+\subsection{Wide bar}
+
+ShorTeX provides the \verb!\widebar! command to typeset a wide bar accent on top of a character (similar to the 
+usual \verb!\widehat! and \verb!\widetilde! commands). Compare to the usual \verb!\bar! and 
+\verb!\overline! commands:
+\[
+	\text{\texttt{widebar}:}\,\, \widebar{A} \qquad \text{\texttt{overline}:} \,\,\overline{A} \qquad \text{\texttt{bar}:} \,\,\bar{A}
+\]
+The code for \verb!\widebar! was taken from \url{https://tex.stackexchange.com/questions/16337/can-i-get-a-widebar-without-using-the-mathabx-package}.
+
+
+
+\subsection{Commenting}
+ShorTeX defines two types of comments that can be used 
```

Comment:
Assuming it's still there, you should mention the `\suppresscomments` command

Reply:
It is gone -- I will find it again, re-add, document